build.gradle

```diff
@@ -24,21 +36,20 @@ allprojects {
     }
 }
 
-if (isReleaseVersion) {
-    apply plugin: "io.github.gradle-nexus.publish-plugin"
-
-    nexusPublishing {
-        repositories {
-            sonatype {
-                def ossUser = System.getenv("SONATYPE_USERNAME") ?: project.hasProperty("sonatypeOssUsername") ? project.sonatypeOssUsername : ''
-                def ossPass = System.getenv("SONATYPE_PASSWORD") ?: project.hasProperty("sonatypeOssPassword") ? project.sonatypeOssPassword : ''
-                def ossStagingProfileId = System.getenv("SONATYPE_STAGING_PROFILE_ID") ?: project.hasProperty("sonatypeOssStagingProfileId") ? project.sonatypeOssStagingProfileId : ''
-                nexusUrl = uri("https://s01.oss.sonatype.org/service/local/")
-                snapshotRepositoryUrl = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
-                username = ossUser
-                password = ossPass
-                stagingProfileId = ossStagingProfileId
-            }
-        }
+dependencyManagement {
+    imports {
+        mavenBom "org.grails:grails-bom:$grailsVersion"
     }
+    applyMavenExclusions false
 }
+

```

Comment:
Please make sure this looks correct, I saw this was done in another PR but I am still a bit unsure about the publishing here.   I wasn't sure if I also need this in another build.gradle file...will comment there as well

Reply:
It matches https://github.com/gpc/fields/blob/6.0.x/build.gradle#L55-L64.  Once we merge we will see if anything errors.  There are occasionally a few tweaks to get the snapshot published.